tests/src/test/scala/whisk/core/cli/test/SequenceMigrationTests.scala

```diff
@@ -55,6 +54,9 @@ class SequenceMigrationTests
     val wsk = new Wsk
     val whiskConfig = new WhiskConfig(WhiskEntityStore.requiredProperties)
     // handle on the entity datastore
+
+    SharedModules.initSharedModules(List(new SharedModule(actorSystem, whiskConfig, logging)))
```

Comment:
actually I think this is no longer required - previously it was instantiating all Spi instances on load of first instance, now it only instantiates impl on access of first instance of that Spi type; removed these calls for now, but if they are needed, I now have a way to add them as you describe.